core/test/functional/routes/api/public_api_spec.js

```diff
@@ -503,4 +504,70 @@ describe('Public API', function () {
                 done();
             });
     });
+
+    it('fetch the most recent post, then the prev, then the next should match the first', function (done) {
+        function createFilter(publishedAt, op) {
+            // This line deliberately uses double quotes because GQL cannot handle either double quotes
+            // or escaped singles, see TryGhost/GQL#34
+            return encodeURIComponent("published_at:" + op + "'" + publishedAt + "'");  // jscs:ignore
+        }
+
+        request.get(testUtils.API.getApiQuery(
```

Comment:
Did you know that [supertest](https://github.com/visionmedia/supertest#example) offers using `.then` chaining?

Reply:
I did not

Reply:
I discovered it last week while working on a service. I was 🎊.